pkg/scorecard_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pkg
+
+import (
+	"testing"
+)
+
+func TestRepoURL_Set(t *testing.T) {
+	type fields struct {
+		Host  string
+		Owner string
+		Repo  string
+	}
+	type args struct {
+		s string
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		wantErr bool
+	}{
+		{
+			name: "Valid http address",
+			fields: fields{
+				Host:  "github.com",
+				Owner: "kubeflow",
```

Comment:
nit: pick a different owner and repo to make sure we parse them right.

Reply:
Do you want to check if the parsed value is the expected value?

Reply:
yes right, just want to make sure owner and repo part are parsed correctly too.

Reply:
Took care of it.